### PR TITLE
Devenv

### PR DESCRIPTION
**Pull request by saimonmoore**
I tried this yesterday. It mostly went quite smoothly. I think it should be merged and then we can continue to iterate on it. Going to add some bits and pieces in another branch and write a bit of documentation.

### CODE REVIEW

File: inventory/hosts [hunk 1]
```diff
@@ -74,4 +72,3 @@ ie
 in
 uk
 us
-de
```
Comment (sauloperez) [inline inventory/hosts:77]: I'm offended :trollface:

Comment (saimonmoore) [reply inventory/hosts:77]: lol

**Review by danypr92 — DISMISSED** [review]

File: playbooks/development.yml [hunk 2]
```diff
@@ -44,6 +44,10 @@
     - role: dbserver # Set up database server and user for the app.
       tags: dbserver
       db_user_roles: SUPERUSER,CREATEDB
+
+    - role: devloper # Handy tools for developers
+      tags: devloper
```
Comment (sauloperez) [inline playbooks/development.yml]: is this intended? wouldn't it be better as `developer`?

Comment (saimonmoore) [reply playbooks/development.yml]: it's a pun...@devloper

**Review by sauloperez — APPROVED** [review]

**Review by danypr92 — APPROVED** [review]

**Pull request merged**